thirdparty/runtime_environments_test.go

```diff
@@ -0,0 +1,17 @@
+package thirdparty
+
+// TODO: Uncomment when DEVPROD-6983 is resolved. Right now, the API does not work on task hosts.
+// func TestGetImageNames(t *testing.T) {
```

Comment:
Winston was able to get this working after deploying a new AMI; I'd suggest trying to run it again!

Reply:
It worked!